packages/e2e/features/ui/hooks/query-datastore.feature

```diff
@@ -0,0 +1,24 @@
+Feature: useDataStore hook
+
+  The useDataStore hook works correctly
```

Comment:
I'd add more information here:

E2e Feature header usually has some information about the feature itself (e.g. `useDataStore` does this, that, and that) more than just a spec name.

Then scenarios below will provide more specific spec info.